proof/crefine/RISCV64/Detype_C.thy

```diff
@@ -1025,10 +1012,11 @@ proof -
 qed
 
 lemma cnotification_relation_restrict:
-  assumes vuc: "s \<turnstile>' capability.UntypedCap d ptr bits idx"
-  and    invs: "invs' s"
-  and      rl: "\<forall>(p :: machine_word) P. ko_wp_at' P p s \<and> (\<forall>ko. P ko \<longrightarrow> live' ko) \<longrightarrow> p \<notin> {ptr..ptr + 2 ^ bits - 1}"
-  and    meps: "map_to_ntfns (ksPSpace s) p = Some ntfn"
+  assumes  vuc: "s \<turnstile>' capability.UntypedCap d ptr bits idx"
+  and     invs: "invs' s"
+  and sym_refs: "sym_refs (state_refs_of' s)"
+  and       rl: "\<forall>(p :: machine_word) P. ko_wp_at' P p s \<and> (\<forall>ko. P ko \<longrightarrow> live' ko) \<longrightarrow> p \<notin> {ptr..ptr + 2 ^ bits - 1}"
```

Comment:
I know you've just swapped the line, but if we're touching it anyway, we could remove the unnecessary parentheses and type annotation for `p`.